### DISCUSSION TITLE
Error installing the Android Studio - Please Help! javax.net.ssl.SSLException: Tag mismatch!

### DISCUSSION THREAD

**Discussion by Samwellion** [strimzi/strimzi-kafka-operator#10310] [Q&A]
I can not get Android Studio to install on my computer as I am getting the following error when installing SDK components (see screenshot). I have tried installing/uninstalling and installing again. I have tried installing as Admin and User. 
![Screenshot 2024-06-16 160019](https://github.com/strimzi/strimzi-kafka-operator/assets/171516222/050a8342-7495-43dc-9fbc-eb82369b8e8f)

**Comment by scholzj** [comment]
Wrong repository?

**Comment by Samwellion** [comment]
Yes, I am new and a bit clueless. Where should I post this to get help?

**Reply by scholzj** [reply]
This discussion is about the Strimzi project. Your problem seems to be with Android Studio? So you should ask it somewhere at the Android Studio project? But I never used Android Studio, so no idea where that would be.

**Reply by Samwellion** [reply]
Thanks!
